### PR TITLE
Add hexo-fun-time plugin

### PR DESCRIPTION
**Pull request by Drunkman**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [x] I want to publish my plugin on Hexo official website.
    - [x] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [x] `link` URL is correct.
- [ ] Others (Update, fix, translation, etc...)

### CODE REVIEW

File: source/_data/plugins.yml [hunk 1]
```diff
@@ -2277,3 +2277,9 @@
     - options
     - parameters
     - customization
+- name: hexo-fun-time
+  description: A plugin for hexo. Insert fun time into posts.
```
Comment (curbengh) [inline source/_data/plugins.yml]: Suggest "A tag plugin to insert random Mandarin jokes in a post"

Comment (Drunkman) [reply source/_data/plugins.yml]: The author can use one tag `{% funTime %}` to insert one joke. 
Maybe generating joke according to header (h1, h2, ...) in future, but each part has just one joke.

**Comment by curbengh** [comment]
I got this in default option,

``` html
<form><fieldset style="width: 400px; padding: 10px; margin: 10px auto; border: #000 2px dotted; border-radius: 10px; "><legend style="font-size: 24px; font-style: italic; text-shadow: 0 0 0.2em #8F7, 0 0 0.2em #8F7; ">开心一刻</legend><div style>[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object],[object Object]</div></fieldset></form>
```

**Comment by Drunkman** [comment]
Sorry, it is my fault. Has fixed it in latest version (0.0.4)

**Pull request merged**